cpu/native/include/socket_zep.h

```diff
@@ -10,6 +10,9 @@
  * @defgroup    drivers_socket_zep  Socket-based ZEP
  * @ingroup     drivers_netdev
  * @brief       UDP socket-based IEEE 802.15.4 device over ZEP
+ *
+ * @see net_zep
```

Comment:
This doesn't resolve and requires a `@ref`. Better: `@see @ref net_zep for protocol definitions`.

Reply:
indeed, will fix